subnet/etcd/local_manager.go

```diff
@@ -347,6 +347,10 @@ func (m *LocalManager) WatchLeases(ctx context.Context, cursor interface{}) (Lea
 	}
 }
 
+func (m *LocalManager) SetNodeNetworkUnavailableFalse(ctx context.Context) error {
+	return nil
```

Comment:
Could you return an error saying: unimplemented? For example: https://github.com/flannel-io/flannel/blob/master/subnet/kube/kube.go#L458

Reply:
look at new update